cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1743,21 +1783,22 @@ display.page.search.tab = statistics
 display.page.search.mode = fast
 request.ui_dispatch_app = cyences_app_for_splunk
 request.ui_dispatch_view = search
-search = `cs_gws` sourcetype="gws:reports:drive" NOT email IN ("", "null") _index_earliest=-61m@m _index_latest=-1m@m \
+search = `cs_gws` sourcetype="gws:reports:drive" "event.type"=acl_change visibility IN ("shared_externally","people_with_link") _index_earliest=-61m@m _index_latest=-1m@m \
 | rex field=owner "[^@]+@(?<src_domain>[^@]+)" \
 | rex field=email "[^@]+@(?<dest_domain>[^@]+)" \
+| where isnotnull(src_domain) AND isnotnull(dest_domain) \
 | search src_domain IN `cs_email_user_domain` AND NOT dest_domain IN `cs_email_user_domain` \
-| stats values(doc_title) as doc_title, values(doc_type) as doc_type, values(email) as dst_email_list, values(visibility) as visibility, count, min(_time) as firstTime, max(_time) as lastTime by owner, ipAddress \
 | `cs_user_privilege_mapping(owner)` \
+| stats values(doc_title) as doc_title, values(doc_type) as doc_type, values(event.name) as event.name, values(email) as dst_email_list, values(visibility) as visibility, count, min(_time) as firstTime, max(_time) as lastTime by owner \
```

Comment:
As we are doing by owner. keep **| `cs_user_privilege_mapping(owner)`**  after the stats command otherwise we have to add it into the stats